apps/src/weblab2/Weblab2View.tsx

```diff
@@ -20,11 +21,12 @@ import VerticalLayout from './layout/VerticalLayout';
 const weblabLangMapping: {[key: string]: LanguageSupport} = {
   html: html(),
   css: css(),
+  js: javascript(),
 };
 
 const defaultConfig: ConfigType = {
   languageMapping: weblabLangMapping,
-  editableFileTypes: ['html', 'css'],
+  editableFileTypes: ['html', 'css', 'js'],
```

Comment:
Do we want to add `.js` to `BACKPACK_PROGRAM_FILE_TYPES` in `files_api.rb` so we skip the profanity check for libraries since backpack files can't be shared with others?

Reply:
oh right!  Do we want to add a condition then to exclude .js files from `pythonlab`? We have easy access to the `project_type`.

Reply:
Oh yeah - `weblab2`. Thanks!

Reply:
https://codedotorg.atlassian.net/browse/CT-1242